pkg/crinit/util/util.go

```diff
@@ -170,6 +170,86 @@ func UpdateKubeconfig(pathOptions *clientcmd.PathOptions, name, endpoint,
 	return nil
 }
 
+// DeleteKubeconfigEntry helper to delete the kubeconfig file entry based on input
+// parameters.
+func DeleteKubeconfigEntry(out io.Writer, pathOptions *clientcmd.PathOptions, name,
+	kubeConfigPath string, dryRun, ignoreErrors bool) error {
+
+	pathOptions.LoadingRules.ExplicitPath = kubeConfigPath
+	kubeconfig, err := pathOptions.GetStartingConfig()
+	if err != nil {
+		return err
+	}
+
+	kubeconfigFile := pathOptions.GetDefaultFilename()
+	if pathOptions.IsExplicitFile() {
+		kubeconfigFile = pathOptions.GetExplicitFile()
+	}
+
+	if dryRun {
+		return nil
+	}
+
+	// If we are not going to ignore errors, then return an error immediately
+	// on the first error encountered. If ignoring errors, then only output
+	// errors when verbose logging is enabled. If it turns out that all three:
+	// context, cluster, and authinfo were not found, then return an error as
+	// there is no point in updating the kubeconfig and the ignore errors flag
```

Comment:
What do you mean "the ignore errors flag will also be checked by the caller."?

Reply:
I updated the comment. Let me know if it is clear.